homeassistant/components/calendar/__init__.py

```diff
@@ -10,7 +10,7 @@
 import re
 
 from homeassistant.components.google import (
-    CONF_OFFSET, CONF_DEVICE_ID, CONF_NAME)
+    CONF_DEVICE_ID, CONF_NAME)
```

Comment:
For bonus points, you could now make the Google Calendar platform import its `CONF_OFFSET` from this module…